src/stream/src/executor/sink.rs

```diff
@@ -302,18 +300,17 @@ impl<F: LogStoreFactory> SinkExecutor<F> {
             .filter_map(|(idx, column)| (!column.is_hidden).then_some(idx))
             .collect_vec();
 
-        let log_reader = log_reader.transform_chunk(move |chunk| {
-            sink_metrics
-                .connector_sink_rows_received
-                .inc_by(chunk.cardinality() as u64);
```

Comment:
Is this metrics been missed now? cc @wenym1

Reply:
I find that the metrics in Dashboard is always 0.
<img width="742" alt="image" src="https://github.com/risingwavelabs/risingwave/assets/43447882/83ba47f8-64e2-47b0-b617-6e8e70f66ed6">

Reply:
Yes. This metric has been removed in this PR. Let's add it back https://github.com/risingwavelabs/risingwave/pull/12812